Resources/doc/format_listener.rst

```diff
@@ -124,6 +124,35 @@ The matched version is set as a Request attribute with the name ``version``,
 and when using JMS serializer it is also set as an exclusion strategy
 automatically in the ``ViewHandler``.
 
-See `this JMS Serializer article`_ for details.
+To make the version mechanism working :
+
+1 - The client must pass the requested version in his header like this :
+
+.. code-block:: yaml
+
+    Accept:application/json;version=1.0
```

Comment:
you should reference http://symfony.com/doc/master/bundles/FOSRestBundle/format_listener.html#media-type-version-extraction as the format is actually configurable